test/blackbox-tests/test-cases/foreign-objects-lib.t

```diff
@@ -0,0 +1,106 @@
+----------------------------------------------------------------------------------
+Test suite for (foreign_objects ...) field.
+
+----------------------------------------------------------------------------------
+* (foreign_objects ...) is unavailable before Dune 3.5.
+
+  $ echo "(lang dune 3.4)" > dune-project
+  $ mkdir -p lib
+
+  $ touch calc.opam
+
+  $ cat >lib/dune <<EOF
+  > (library
+  >  (public_name calc)
+  >  (foreign_objects add mul))
+  > EOF
+
+  $ dune build
+  File "lib/dune", line 3, characters 1-26:
+  3 |  (foreign_objects add mul))
+       ^^^^^^^^^^^^^^^^^^^^^^^^^
+  Error: 'foreign_objects' is only available since version 3.5 of the dune
+  language. Please update your dune-project file to have (lang dune 3.5).
+  [1]
```

Comment:
```suggestion
  Error: 'foreign_objects' is only available since version 3.5 of the Dune
  language. Please update your ``dune-project`` file to have (lang dune 3.5).
  [1]
```

Although I can understand keeping them the way they are if this is the way the error presents

Reply:
Yeh, this is just the error produced by `Dune_lang.Syntax.since`